Gemfile.lock

```diff
@@ -417,7 +417,7 @@ GEM
       sparql (~> 1.99)
       sparql-client (~> 1.99)
     little-plugger (1.1.4)
-    logger (1.2.8)
+    logger (1.2.8.1)
```

Comment:
Seems that logger 1.2.8 was removed from Rubygems, but this version does exist. If you already had logger 1.2.8 installed you would be fine bundling, but I did not have this version for whatever reason.